pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
If you look at the comments in `test_categorical.py` (which I now understand why they were included), you'll see that in fact the old pickles do have the `levels` attribute passed in, as far back as `test_pickle_v0_14_1` - is a test in `0.18.1` for `levels` necessary then?